Update GitHub example

lucascurti opened the pull request:
- `.npmrc` is no longer needed
- Updated instructions
- updated `package.json` versions so it works independently of the parent project.
- updated `serverBuildDirectory` since it wasn't working with the old path

sergiodxa commented on examples/github/remix.config.js:
```diff
@@ -1,10 +1,10 @@
   browserBuildDirectory: "public/build",
   publicPath: "/build/",
-  serverBuildDirectory: "server/build",
+  serverBuildDirectory: "build",
```
Why change this? I prefer to keep the server build inside the server directly, it should work correctly.

lucascurti commented:
@sergiodxa Sorry for the delay on this. Updated the code as per your comments and and updated so it works with v3.0.0. Let me know if you prefer to have this example in `remix-auth-github` and I'll do a PR on that repo.

sergiodxa commented:
> @sergiodxa Sorry for the delay on this. Updated the code as per your comments and updated it so it works with v3.0.0. Let me know if you prefer to have this example in `remix-auth-github` and I'll do a PR on that repo.

Yeah it will be better to have the example of the strategy directly in the strategy repository instead.
